cdrage/podman-desktop
# fix: store error for build

fix: store error for build

### What does this PR do?

When building, we did not previously store the error.

This fixes the Task notification by storing the error when we encounter
it within build-image-task.ts

Added tests

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/3026

### How to test this PR?

1. Purposely build a failing Dockerfile
2. Check Tasks that it shows the error



Signed-off-by: Charlie Drage <charlie@charliedrage.com>
